app/ide-desktop/eslint.config.js

```diff
@@ -34,7 +34,7 @@ const DEFAULT_IMPORT_ONLY_MODULES =
 const OUR_MODULES = 'enso-.*'
 const RELATIVE_MODULES =
     'bin\\u002Fproject-manager|bin\\u002Fserver|config\\u002Fparser|authentication|config|debug|detect|file-associations|index|ipc|log|naming|paths|preload|project-management|security|url-associations|#\\u002F.*'
-const ALLOWED_DEFAULT_IMPORT_MODULES = `${DEFAULT_IMPORT_ONLY_MODULES}|postcss|${RELATIVE_MODULES}`
+const ALLOWED_DEFAULT_IMPORT_MODULES = `${DEFAULT_IMPORT_ONLY_MODULES}|postcss|ajv\\u002Fdist\\u002F2020|${RELATIVE_MODULES}`
```

Comment:
Out of curiosity: what is this setting?

Reply:
it's a whitelist of modules where we can do `import TheDefaultImport from 'module'` rather than `import * as theModuleNamespace from 'module'`. usually it's a limitation of the module simply not exposing the default export as a named export as well